daisy_workflows/linux_common/utils/common.py

```diff
@@ -721,3 +721,14 @@ def GetForwardingRuleIP(self, name):
         project=self.project, region=self.region, forwardingRule=name)
     response = request.execute()
     return response[u'IPAddress']
+
+
+@RetryOnFailure
```

Comment:
How does RetryOnFailure work? Does it give up after certain number of retries? Is there an exponential back off?

Reply:
Here's the decorator. It does linear backoff (1.5 coefficient), stopping after five minutes.

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/aaf4e0b12c637213702aefa762e954eb99a771ac/daisy_workflows/linux_common/utils/common.py#L36

Reply:
It's actually stopping after 15 minutes, as we already discussed in person. 5 minutes sounds more reasonable for this use case.